pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -808,8 +816,7 @@ func (vs *VSphere) LoadBalancer() (cloudprovider.LoadBalancer, bool) {
 
 // Zones returns an implementation of Zones for vSphere.
 func (vs *VSphere) Zones() (cloudprovider.Zones, bool) {
-	glog.V(1).Info("The vSphere cloud provider does not support zones")
-	return nil, false
+	return vs, true
```

Comment:
As we are discussing if vcp config should be required on the nodes, maybe we should return `false` here if there is no vcenter configured?

Reply:
you mean no `vcp.conf` file or no `[labels]` field in `vcp.conf` file? 
If it's the second case, since the kubelet code will detect whether the `zone` and `region` is `""` or not. If it's empty, the kubelet code will handle this. So we can just `return vs, true` here.

Reply:
I was thinking no vcenter == `if vs.cfg.Workspace.VCenterIP == "": return false`

Reply:
If there is no `vcp.conf` file, `Zones()` maybe even not called since `get node` return `No resources found`

Reply:
I see. one more concern, Should we also apply this check to other functions/interfaces in `vsphere.go` which they use `vcp.conf` file?